tools/config-generator/README.md

```diff
@@ -0,0 +1,11 @@
+# config-generator
+
+config-generator is a tool that takes a meta config file (e.g. ../../config/prow/config_knative.yaml)
```

Comment:
Thank you for this README, I think we should also mention where templates are. Related, should we move templates here?

Reply:
Done.